.github/workflows/haskell-ci.yml

```diff
@@ -21,18 +21,15 @@ jobs:
     strategy:
       matrix:
         ghc-version: [latest,
-                      9.10.1,
+                      9.12.2,
+                      9.10.2, 9.10.1,
                       9.8.4, 9.8.2, 9.8.1,
                       9.6.6, 9.6.5, 9.6.4, 9.6.3, 9.6.2, 9.6.1,
                       9.4.8, 9.4.7, 9.4.6, 9.4.5, 9.4.4, 9.4.3, 9.4.2, 9.4.1,
                       9.2.8, 9.2.7, 9.2.6, 9.2.5, 9.2.4, 9.2.3, 9.2.2, 9.2.1,
                       9.0.2, 9.0.1,
                       8.10.7, 8.10.6, 8.10.5, 8.10.4, 8.10.3, 8.10.2, 8.10.1,
-                      8.8.4, 8.8.3, 8.8.2, 8.8.1,
-                      8.6.5, 8.6.4, 8.6.3, 8.6.2, 8.6.1,
-                      8.4.4, 8.4.3, 8.4.2, 8.4.1,
-                      8.2.2,
-                      8.0.2]
```

Comment:
Why can't you test GHCs older than 8.10?

Reply:
The more official information (in the .cabal file) is `base >=4.3`, this means you are supporting GHC 7.0 and up.
I think the `tested-with` should be in sync with both CI _and_ the `base` constraint, otherwise there are untested/untried build plans.
So you aim to drop all of GHC < 8.10?